### DISCUSSION TITLE
brackets

### DISCUSSION THREAD

**Discussion by ghost** [brackets-cont/brackets#156] [General]
How to uninstall Brackets extensions.
I installed too many extension and I am overwhelmed.
Help me find the basic version please
Thank you

**Comment by abose** [comment]
Can you answer the following :
1. which OS are you using?
2. Do you want to factory reset brackets?

**Reply by ghost** [reply]
MacOs Monterey
Version 12.1

**Reply by ghost** [reply]
I want to reinstall brackets with the basic version because I'm a beginner.
I can change code editor simply but I want to use Brackets.
I want to reset everything
Thank you

**Reply by Pulkit-Pandey-09** [reply]
Hi!
To reset, try to delete app data folder.

**Reply by abose** [reply]
To reset brackets in mac, provided you have installed brackets in the applications directory, follow these steps:
* Open `terminal` app in mac
* Type this command: `rm -rf ~/Library/Application Support/Brackets`
@tecinaadje

**Reply by ghost** [reply]
I went to terminal and typed the command:
~/Library/Application Support/Brackets
nothing happens (see attached [screenshot)[](url)]([url]([url](url))
![Capture](https://user-images.githubusercontent.com/94017981/154671329-37cee632-c00b-4c26-8626-2ca9415586ec.jpg)
)

**Reply by ghost** [reply]
If I delete the affected brackets folder and reinstall Brackets, the exact same thing happens.
All extensions reappear

**Reply by ghost** [reply]
After the following command: rm -rf ~/Library/Application Support/Brackets
 I found the basic version.
thank you very much
